dynawo/CMakeLists.txt

```diff
@@ -88,6 +89,7 @@ set(CXX11_ENABLED ON CACHE BOOL "")
 if(MSVC)
   add_definitions(-D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS -D_WIN32_WINNT=0x0601)
   add_definitions(-DLANG_CXX11)
+  add_definitions(-bigobj)
```

Comment:
Did you encounter the maximum size issue?

Reply:
No but it prevents any issue in the future

Reply:
The recommendation is to use it on a need-to basis as it may create issue with old linkers. So I propose to not add it unless we need it.

Reply:
ok